app/main.py

```diff
@@ -2,0 +1,30 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
```

Comment:
According to the checklist, make sure you use the double quotes everywhere. `print(f"Animal __init__ called with: {name}, health: {health}")`.